bgpd/bgp_lcommunity.c

```diff
@@ -585,8 +592,8 @@ void bgp_compute_aggregate_lcommunity(struct bgp_aggregate *aggregate,
 	/* Create hash if not already created.
 	 */
 	if (aggregate->lcommunity_hash == NULL)
-		aggregate->lcommunity_hash = hash_create(
```

Comment:
it would be better do not touch the parts you don't need to touch, reference: https://github.com/FRRouting/frr/blob/master/doc/developer/workflow.rst 

>Whitespace changes in untouched parts of the code are not acceptable in patches that change actual code. To change/fix formatting issues, please create a separate patch that only does formatting changes and nothing else.